frontend/src/queries/Project.ts

```diff
@@ -37,6 +53,26 @@ export function useProjectsQuery(): UseQueryReturnType<Project[], Error> {
     });
 }
 
+export function useGroupsQuery(
+    projectId: Ref<number | undefined>
+): UseQueryReturnType<Group[], Error> {
+    return useQuery<Group[], Error>({
+        queryKey: computed(() => GROUPS_QUERY_KEY(projectId.value!)),
+        queryFn: () => getGroups(projectId.value!),
+        enabled: computed(() => projectId.value !== undefined),
+    });
+}
+
+export function useSubmissionsQuery(
+    projectId: Ref<number | undefined>
+): UseQueryReturnType<Submission[], Error> {
+    return useQuery<Submission[], Error>({
+        queryKey: computed(() => SUBMISSIONS_QUERY_KEY(projectId.value!)),
+        queryFn: () => getSubmissions(projectId.value!),
+        enabled: computed(() => projectId.value !== undefined),
```

Comment:
```suggestion
        enabled: !!toValue(projectId),
```
volgens de nieuwe stijl

Reply:
```suggestion
        enabled: () => !!toValue(projectId),
```
Dit zou de juiste aanpassing zijn